tests/index.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../index');
+
+chai.should();
```

Comment:
❓ What is this doing?

Reply:
You can either use assert or should in chai. So, chai.should() is used to construct assertions

Reply:
🙂 You can also use `expect`. What I wanted to say is commenting the code will be helpful for other developers unfamiliar with Chai, when reading the code.